assets/js/modules/analytics-4/utils/extract-dashboard-data.js

```diff
@@ -106,7 +106,8 @@ export function extractAnalytics4DashboardData(
 
 	// Pad rows to 2 x number of days data points to accommodate new accounts.
 	if ( days * 2 > rowLength ) {
-		const date = new Date();
+		// Don't use the getReferenceDate selector here since there is no need in it.
```

Comment:
The IB says that we can't use the reference date here without refactoring, but not that we _shouldn't_. It actually probably makes sense to use it here, right? We should either explain why we _shouldn't_ use the reference date here, or add a TODO & file a follow-up issue.

Reply:
Ok, updated to use the reference date.